src/webhooks/__tests__/registry.test.ts

```diff
@@ -653,6 +690,181 @@ describe('shopify.webhooks.getTopicsAdded', () => {
   });
 });
 
+describe('dual webhook registry instances', () => {
+  let httpWebhookRegistry: HttpWebhookRegistry;
+  let httpWebhookRegistry2: HttpWebhookRegistry;
+
+  const testConfig2: ConfigParams = {
+    apiKey: 'test_key_2',
+    apiSecretKey: 'dogs are cute too',
+    scopes: ['test_scope2'],
+    hostName: 'test_host_name_2',
+    hostScheme: 'https',
+    apiVersion: LATEST_API_VERSION,
+    isEmbeddedApp: true,
+    isPrivateApp: false,
+    sessionStorage: new MemorySessionStorage(),
+    customShopDomains: undefined,
+    billing: undefined,
+  };
+
+  const shopify2 = shopifyApi(testConfig2);
+
+  let webhookHandlerCalled = false;
+  async function webhookHandler(
+    _topic: string,
+    _shopDomain: string,
+    _body: string,
+  ): Promise<void> {
+    webhookHandlerCalled = true;
+  }
+
+  let webhookHandler2Called = false;
+  async function webhookHandler2(
+    _topic: string,
+    _shopDomain: string,
+    _body: string,
+  ): Promise<void> {
+    webhookHandler2Called = true;
+  }
+
+  beforeEach(async () => {
+    httpWebhookRegistry = (shopify.webhooks as ShopifyWebhooksTesting).testing
```

Comment:
We should just create new instances of `shopify` (which we already do) so we don't need to clean these up every time.